.gitpod.Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM gitpod/workspace-full
+
+RUN npm i gulp-cli -g
```

Comment:
As gulp is already part of project dependencies, this is not needed in here.
Instead, install the right NodeJS version with NVM.

```suggestion
# Setup NVM
COPY .nvmrc $HOME
RUN bash -c "source $HOME/.nvm/nvm.sh && nvm install && nvm use"
```